USER_GUIDE.md

```diff
@@ -1,7 +1,166 @@
 # User Guide (1.3.0-rc)
 Primitive guide for users 1.3.0 and above (need to expand)
 
-# Yaml configuration deployment
+## Applying the Plugin
+For both _standard_ and _flexible_ applications, include the plugin in your buildscript
+
+```
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath "com.google.cloud.tools:appengine-gradle-plugin:<version>"
+  }
+}
+
+apply plugin: "com.google.cloud.tools.appengine"
+```
+
+The [Cloud SDK](https://cloud.google.com/sdk) is required for this plugin to 
+function. Download and install it before running any tasks.
+
+---
+
+## App Engine Standard
+The plugin will include App Engine standard features if you include an `appengine-web.xml`
+in `src/main/webapp/WEB-INF/`, otherwise it will assume it is an [App Engine flexible][#app-engine-flexible] application.
+
+### Tasks
+For App Engine standard, the plugin exposes the following tasks :
+
+#### Local Run
+
+| Task             | Description |
+| ---------------- | ----------- |
+| `appengineRun`   | Run the application locally. |
+| `appengineStart` | Start the application in the background. |
+| `appengineStop`  | Stop a running application. |
+
+#### Deployment
+
+| Task                      | Description |
+| ------------------------- | ----------- |
+| `appengineStage`          | Stage an application for deployment. |
+| `appengineDeploy`         | Deploy an application. |
+| `appengineDeployCron`     | Deploy cron configuration. |
+| `appengineDeployDispatch` | Deploy dispatch configuration. |
+| `appengineDeployDos`      | Deploy dos configuration. |
+| `appengineDeployIndex`    | Deploy datastore index configuration. |
+| `appengineDeployQueue`    | Deploy queue configuration. |
+
+#### Other
+
+| Task                         | Description |
+| ---------------------------- | ----------- |
+| `appengineShowConfiguration` | Print out the appengine standard gradle plugin configuration |
+
+### Configuration
+Once you've configured `gcloud` in the Cloud SDK, no gradle configuration should be needed to run
```

Comment:
Also why is that no further configuration should be needed? Even if we assume they use the default locations, they may want to customize how run/deploy is executed. I understand we want to say here, that if you use the defaults, you can run/deploy the app, but maybe we should say that if you're happy with the defaults (we don't show them here, so referencing them may not be a good idea), you should be able to run/deploy the app. If you need further customization you can blah-blah-blah.